Added device support for DEVICE_C6603_1275_1562_10_1_1_A_1_253 (Sony Xperia Z)

wphungsuk opened the pull request:
Added device support for DEVICE_C6603_1275_1562_10_1_1_A_1_253 (Sony Xperia Z)

fi01 committed:
Initial commit

fi01 committed:
Import detect_device.[ch]

fi01 committed:
Support SH-04E (01.00.03)

fi01 committed:
Support N-05E (A1000311)

fi01 committed:
Support SO-04D and SO-05D

fi01 committed:
Support HTL22 (1.05.970.1) and HTX21 (1.20.971.1)

fi01 committed:
Check rom version by device specific property

fi01 committed:
Use ROM version instead of display id for HTC devices

fi01 committed:
Support LT26w 1266-3278 (6.2.B.0.200)

fi01 committed:
Support HTL22 (1.07.970.4)

fi01 committed:
Support SO-01E (9.1.C.0.473)

DooMLoRD committed:
added LT26i 1257-8080 6.2.B.0.11

DooMLoRD committed:
added C6603 1270-6704 10.1.1.A.1.307

DooMLoRD committed:
added C5302 1272-1092 12.0.A.1.284

fi01 committed:
Merge pull request #1 from DooMLoRD/master

support for Xperia Z & S

fi01 committed:
Use prop ro.semc.version.sw to detect Xperia LT26w

fi01 committed:
Merge pull request #2 from DooMLoRD/temp

added C5302 1272-1092 12.0.A.1.284

wphungsuk committed:
Added support for device Sony Xperia Z (DEVICE_C6603_1275_1562_10_1_1_A_1_253)

wphungsuk committed:
Added support for device LG Optimus G E975 (DEVICE_LG_E975)

hiikezoe commented:
This repository has been moved to https://github.com/android-rooting-tools/android_run_root_shell .
Please send pull request to the new repository. I am sorry for the inconvenience.

Thank you,

wphungsuk committed:
Added support for Sony Xperia Z (DEVICE_C6603_1275_1562_10_1_1_A_1_253) and LG Optimus G E275 (DEVICE_LG_E275)

wphungsuk commented:
I have just now sent pull request for the devices to https://github.com/fi01/android_device_database

wphungsuk commented:
I have sent pull request to the new repository on https://github.com/android-rooting-tools/android_run_root_shell

fi01 committed:
Support SC-01E LJ3
Support SC-04E MDI and MF1
Adjust build version for SCL21

fi01 committed:
Support SGP312 10.1.C.0.370
Support SO-02E 10.1.D.0.343
Support SO-03E 10.1.E.0.265
Support SO-03E 10.1.E.0.269
Support SO-04E 10.1.1.D.0.179
Support SOL22 10.2.F.3.43

wphungsuk committed:
Added check property value for LG Optimus G E-975 (DEVICE_LG_E975_V10e)

fi01 committed:
Sort device list

DooMLoRD committed:
added support for Xperia ZL (C6503) - 10.3.A.0.423 - 1266-7597

wphungsuk committed:
Removed merge conflicts

wphungsuk committed:
Merge branch 'master' of https://github.com/wphungsuk/android_run_root_shell

DooMLoRD committed:
added support for Xperia Tablet Z (SGP321) - 10.1.1.A.1.307

DooMLoRD committed:
added support for Xperia Tablet Z (SGP311) - 10.1.C.0.370

DooMLoRD committed:
no need to worry about customizations, seems that for a particular firmware all customizations have the same zImage

DooMLoRD committed:
add compatibility with device variants, seems that for a particular firmware all closely related device variants have the same zImage

DooMLoRD committed:
added support for Xperia T (LT30p) 9.1.A.1.141

wphungsuk committed:
Merge branch 'master' of https://github.com/android-rooting-tools/android_device_database

Commit:
add LT29i 9.1.B.1.67

DooMLoRD committed:
Merge pull request #2 from sj8023ld/master

Add LT29 9.1.B.1.67

fi01 committed:
Add FJL21 V23R39X

fi01 committed:
Add FJL21 V37R47A

fi01 committed:
Added LT29i 9.1.B.0.411

DooMLoRD committed:
added support for Xperia SL (LT26ii) 6.2.B.0.211

DooMLoRD committed:
added support for Xperia Acro S (LT26w) 6.2.B.0.211

DooMLoRD committed:
added support for Xperia P (LT22i) 6.2.A.1.100

DooMLoRD committed:
added support for Xperia Go (ST27i/a) 6.2.A.1.100 (thanks to the_laser)

DooMLoRD committed:
added support for Xperia T (LT30p) 9.1.A.1.142 (thanks to the_laser)

DooMLoRD committed:
added support for Xperia V (LT25i) 9.1.A.1.140/142 (thanks to the_laser)

DooMLoRD committed:
added support for Google Nexus 4 (JDQ39)

fi01 committed:
Added SOL21 9.0.F.0.226

cofface committed:
added support for Galaxy Nexus (JOP40C)

added support for Galaxy Nexus (JOP40C)

cofface committed:
added support for Galaxy Nexus (JOP40C)

added support for Galaxy Nexus (JOP40C)

DooMLoRD committed:
Merge pull request #3 from cofface/patch-1

added support for Galaxy Nexus (JOP40C) [part 1]

DooMLoRD committed:
Merge pull request #4 from cofface/patch-2

added support for Galaxy Nexus (JOP40C) [part 2]

wphungsuk committed:
Merge branch 'master' of https://github.com/android-rooting-tools/android_device_database

Commit:
Add C5306_12.0.A.1.257

Commit:
Add C5303 12.0.A.1.257

DooMLoRD committed:
Merge pull request #5 from sj8023ld/master

Add C5302 C5303 M35h

wphungsuk committed:
Synched changes with remote repo

wphungsuk committed:
Merge branch 'master' of https://github.com/android-rooting-tools/android_device_database
